### PR TITLE
Add growingio analytics

### PR DESCRIPTION
**Pull request by LancelotLewis**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

~~Issue Number(s): N/A~~

## What is the new behavior?
Add a analytics service just as `Baidu Analytics` or `Google Analytics`

~~* Screens with this changes: N/A~~
~~* Link to demo site with this changes: N/A~~

### How to use?
In NexT `_config.yml`:
```yml
growingio: [your growingio ID]
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by sli1989** [comment]
The official website need to be added, for example: `See more: https://docs.growingio.com/sdk-integration/sdk-2.x-docs/`

File: _config.yml [hunk 1]
```diff
@@ -476,6 +476,9 @@ pangu: false
 # Baidu Analytics ID
 #baidu_analytics:
 
+# Growingio Analytics ID
+#growingio:
```
Comment (ivan-nginx) [inline _config.yml]: Here need copyright.

File: layout/_third-party/analytics/growingio.swig [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% if theme.growingio %}
+  <!-- GrowingIO Analytics code version 2.1 -->
+  <!-- Copyright 2015-2018 GrowingIO, Inc. More info available at http://www.growingio.com -->
```
Comment (ivan-nginx) [inline layout/_third-party/analytics/growingio.swig]: No need copiright here for better SEO.

File: layout/_third-party/analytics/index.swig [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {% include 'facebook-sdk.swig' %}
 {% include 'vkontakte-api.swig' %}
+{% include 'growingio.swig' %}
```
Comment (ivan-nginx) [inline layout/_third-party/analytics/index.swig]: Seems GA, Baidu and AI bigger services than growingio. So, only after that it must be included.

**Comment by stevenjoezhang** [comment]
Need to create docs here: https://github.com/theme-next/theme-next.org/blob/source/source/docs/third-party-services/statistics-and-analytics.md

**Comment by stevenjoezhang** [comment]
https://github.com/theme-next/theme-next.org/commit/46d87dd3ee071198615b57c6488d05f8258bd770

**Pull request merged**